doc/api/stream.md

```diff
@@ -437,6 +437,15 @@ process.nextTick(() => {
 
 See also: [`writable.cork()`][].
 
+##### writable.writableLength
+<!-- YAML
+added: REPLACEME
+-->
+
+This property contains the number of bytes (or objects) in the queue
+ready to be written. The value provide introspection data regarding
```

Comment:
typo: provides? (for readableLength as well)

Reply:
typo, fixed.